### PR TITLE
feat: Add support to php 8 attributes

### PR DESCRIPTION
**Pull request by djpremier**
Add support to php 8 attributes, keeping compatibility with annotations.

```php
use DH\Auditor\Provider\Doctrine\Auditing\Annotation as Audit;

#[
    Audit\Auditable,
    Audit\Security(
        view: [
            'ROLE_ADMIN',
        ],
    ),
]
class DummyPhp8
{
    #[Audit\Ignore]
    public string $hello = 'hi';
}
```

### CODE REVIEW

**Comment by DamienHarper** [comment]
@djpremier Thanks a lot for this PR, much appreciated! 
One thing though, tests currently fail with Symfony 3.4 because of `"friendsofphp/php-cs-fixer": "^3.0"` which is not compatible with Symfony 3.4.

You may try something like `"friendsofphp/php-cs-fixer": "^2.16|^3.0"`

File: composer.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
   },
   "require-dev": {
     "doctrine/doctrine-migrations-bundle": "^1.3|^2.0",
-    "friendsofphp/php-cs-fixer": "^2.16",
+    "friendsofphp/php-cs-fixer": "^3.0",
```
Comment (DamienHarper) [inline composer.json]: compatibility problem with Symfony 3.4

**Comment by djpremier** [comment]
> @djpremier Thanks a lot for this PR, much appreciated!
> One thing though, tests currently fail with Symfony 3.4 because of `"friendsofphp/php-cs-fixer": "^3.0"` which is not compatible with Symfony 3.4.
> 
> You may try something like `"friendsofphp/php-cs-fixer": "^2.16|^3.0"`

Thanks, done.

**Review by DamienHarper — APPROVED** [review]

**Review by DamienHarper — CHANGES_REQUESTED** [review]
Looks like tests using Symfony 3.4 and PHP 8 fail but as far as I know, PHP 8 is not supported with Symfony 3.4
Tests should not even be run with such a config.

**Comment by djpremier** [comment]
> Looks like tests using Symfony 3.4 and PHP 8 fail but as far as I know, PHP 8 is not supported with Symfony 3.4
> Tests should not even be run with such a config.

I will check.

**Review by DamienHarper — APPROVED** [review]

**Comment by DamienHarper** [comment]
thanks @djpremier

**Comment by djpremier** [comment]
I'm working on this change also for ```@Gedmo```, as soon as possible I'll fix the missing part of the test.

**Pull request merged**